config_examples/template_config.yaml

```diff
@@ -45,24 +43,94 @@ telescope:
         sep: 14.6
         split_core: True
         outriggers: 2
+    # TODO: check how pyuvsim handles beams, see if YAML constructor needs
+    # to be updated/modified.
+    omega_p: !Beam
+        # TODO: add support for specifying directory containing beam file
+        datafile: HERA_H2C_BEAM_MODEL.npz
+        interp_kwargs:
+            interpolator: interp1d
+            fill_value: extrapolate
+            # if you want to use a polynomial interpolator instead, then
+            # interpolator: poly1d
+            # kwargs not accepted for this; see numpy.poly1d documentation
+    Trx: 0.0 # receiver temperature for thermal noise
+defaults:
+    # behavior of specifying default config and some other
+    # parameters (e.g. omega_p, Tsky_mdl) currently unknown
+    default_config: 'h2c'
 systematics:
-    # TODO: fill out what should go in here
-analysis:
-    # TODO: fill out what should go in here
-simulation:
+    # XXX: perhaps change include section of simulation to just list rfi
+    # and include all the specified rfi effects if rfi is listed
+    rfi:
+        # see hera_sim.rfi documentation for details on parameter names
+        rfi_stations:
+            stations: !!null
+        rfi_impulse:
+            chance: 0.001
+            strength: 20.0
+        rfi_scatter:
+            chance: 0.0001
+            strength: 10.0
+            std: 10.0
+        rfi_dtv:
+            freq_min: 0.174
+            freq_max: 0.214
+            width: 0.008
+            chance: 0.0001
+            strength: 10.0
+            strength_std: 10.0
+    sigchain:
+        gains:
+            gain_spread: 0.1
+            dly_rng: (-20, 20)
+        # XXX: the gen_reflection_gains function is somewhat difficult
+        # to work with in a general way at the moment; this should be
+        # addressed in the future, since there currently is not a neat
+        # way of handling this from a configuration file
+        reflections: # TODO: make a YAML constructor for this?
+            amp: !!null
+            dly: !!null
+            phs: !!null
+    crosstalk:
+        # only one of the two crosstalk methods should be specified
+        whitenoise_xtalk:
+            amplitude: 3.0
+        # cross_coupling_xtalk:
+            # the following parameters are all floats, but it's a little
```

Comment:
the problem here is with how the `add_xtalk` method of the `Simulator` class was written. as it currently stands, every baseline gets the same `kwargs` passed to the crosstalk method, so crosstalk will show up in every visibility waterfall at the same delay, with the same phase and amplitude

Reply:
Hmmm, OK that sounds like it needs to be fixed then.

Reply:
It does need to be fixed, but I'd rather fix it in the version 1 release, since there are a few other issues with the `Simulator` class that need to be fixed, and people can just as well use `gen_whitenoise_xtalk` for a quick-and-dirty simulation of crosstalk.

Reply:
OK, I can go with that. I'll make an issue, which can either be addressed independently, or in your v1 PR.